static/js/lib/websocket.js

```diff
@@ -20,6 +20,14 @@ define([
      */
     var WebsocketManager = function (User, settings) {
             this.settings = $.extend(true, {}, this.settings, settings);
+            //Domain name to be changed
+            if(location.host.split(":").length > 1){
+                this.settings.serverUrl = "ws://"+location.hostname+":5000";                
```

Comment:
le serverUrl n'étant plus une option mais généré dynamiquement, il n'y a plus de raison de le passer dans les settings. Un this.serverUrl suffit.